docs/contributing.rst

```diff
@@ -169,9 +213,9 @@ Here, we also want to highlight the location of the html files that will be gene
 
 *The image above highlights the location of the* :blue:`html` *folder (in a red circle) in reference to all of the other files in the repo, along with the location of the* :blue:`index.html` *file (in the red box).*
 
-When you want to look at the handbook locally (once you generate these html files), you will want to open :blue:`index.html`. This will open the home page of the handbook on your internet browser.
+When you want to preview the handbook locally (once you generate these html files), you will want to open :blue:`index.html`. This will open the home page of the handbook on your internet browser. It is important to understand that these .html files are simply a local version of the website for you to preview how things look. Any modifications you make won't be on the real internet version until your pull request has been approved.
 
-*The general workflow for edits*
+*Editing the website*
 --------------------------------
```

Comment:
@emcdevitt This will build, but for aesthetics, can you shorten the dashed lines here so that it aligns with the new title?

Reply:
done!